exercises/anagram/anagram_test.rb

```diff
@@ -3,119 +3,86 @@
 require 'minitest/autorun'
 require_relative 'anagram'
 
-# Test data version:
-# 6a886e0
+# Common test data version: 196fc1a
 class AnagramTest < Minitest::Test
   def test_no_matches
     # skip
-    detector = Anagram.new('diaper')
-    anagrams = detector.match(["hello", "world", "zombies", "pants"])
-    assert_equal [], anagrams
+    assert_equal [], Anagram.new('diaper').match(["hello", "world", "zombies", "pants"]).sort
   end
 
   def test_detects_simple_anagram
     skip
-    detector = Anagram.new('ant')
-    anagrams = detector.match(["tan", "stand", "at"])
-    assert_equal ["tan"], anagrams
+    assert_equal ["tan"], Anagram.new('ant').match(["tan", "stand", "at"]).sort
   end
 
   def test_does_not_detect_false_positives
     skip
-    detector = Anagram.new('galea')
-    anagrams = detector.match(["eagle"])
-    assert_equal [], anagrams
+    assert_equal [], Anagram.new('galea').match(["eagle"]).sort
   end
 
-  def test_detects_multiple_anagrams
+  def test_detects_two_anagrams
     skip
-    detector = Anagram.new('master')
-    anagrams = detector.match(["stream", "pigeon", "maters"])
-    assert_equal ["maters", "stream"], anagrams.sort
+    assert_equal ["maters", "stream"], Anagram.new('master').match(["stream", "pigeon", "maters"]).sort
```

Comment:
4 would probably be best since you can use another line for `expected`

The main issue here is that we want to prioritize educational value over expert comprehension. 

1 line:
```ruby
assert_equal ["gallery", "largely", "regally"], Anagram.new('allergy').match(["gallery", "ballerina", "regally", "clergy", "largely", "leading"]).sort
```
At my day job I would be fine with the 1 line version as everyone knows the test framework and can maintain a consistent test style, and we are not trying to work out how to implement a solution by looking at the tests.

On Exercism I want to make sure that things are clear to people who may not be as familiar with Minitest and even Ruby. 

Multi line:
```ruby
    detector = Anagram.new('allergy')
    anagrams = detector.match(["gallery", "ballerina", "regally", "clergy", "largely", "leading"])
    expected = ["gallery", "largely", "regally"]
    assert_equal expected, anagrams.sort
```

The multi line version uses variable naming to reveals the intention of the individual elements of the one line assertion. Note that the values used for `detector`,`anagrams` and `expected` are all vary by test.
This provides the information one needs to derive the code to pass the test in a clear and consistent manner.